content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,424 @@
+out = Interchange.from_smirnoff(forcefield, topology)
+```
+
+The functions have different names and outputs, but their inputs are identical!
+
+
+## Exporting to simulation engines
+
+There are high-level API points for exporting to particular file formats. The focus of this project,
+for the most part, does not cover run-time options such as the choice of integreator, thermostat,
+and barostat. Discussions are underway on the possibility of providing partial input scripts that
+require some manual edits but include important information contained within the force field (cutoff
+methods and distances, in particular).
+
+```python
+# GROMACS
+out.to_gro("out.gro")
+out.to_top("out.top")
+
+# Amber
+out.to_inpcrd("out.inpcrd")
+out.to_prmtop("out.prmtop")
+
+# LAMMPS
+out.to_lammps("out.lmp")
+
+# OpenMM (`openmm.System`)
+openmm_system = out.to_openmm()
+
+# The OpenFF Toolkit provides direct exports to `openmm.app.Topology`!
+openmm_topology = topology.to_openmm()
+```
+
+Internally, each of these methods call a function somewhere in the `openff.interchange.interop`
+module; accessing them as methods on the core object is meant to make it easier to actually do each
+operation.
+
+For more on the export layer, see a [corresponding
+section](https://openff-interchange.readthedocs.io/en/latest/using/output.html) in the documentation.
+
+
+## Validating exports
+
+There is, unfortunately, not much of a standard in place for ensuring that parametrizing a
+chemical topology with a given force field produces the expected results. For the purposes of this
+work, we will focus on validating the results of exporting ``Interchange`` objects to files and
+objects understood by supported simulation engines. To this end, there are several functions in the
+``openff.interchange.drivers`` module that each take in an ``Interchange`` object, call out to an
+engine a single-point energy calculation, and store the result.
+
+This snippet continues from above, since we have conveniently already done the work of constructing
+our object. Note that these functions are relatively unintelligent at the moment and simply look to
+see if each engine is installed (i.e. if `gmx`, `sander`, etc. are in `$PATH`). These might not run
+on a machine that does not have them installed - feel free to skip lines if you're following along.
+
+```python
+from openff.interchange.drivers.amber import get_amber_energies
+from openff.interchange.drivers.openmm import get_openmm_energies
+from openff.interchange.drivers.gromacs import get_gromacs_energies
+from openff.interchange.drivers.lammps import get_lammps_energies
+```
+
+```python
+print(get_gromacs_energies(out))
+```
+    Energies:
+
+    Bond:          		17.31004524230957 kJ / mol
+    Angle:         		195.2955780029297 kJ / mol
+    Torsion:       		13.520307540893555 kJ / mol
+    Nonbonded:     		None
+    vdW:           		43.82065216824412 kJ / mol
+    Electrostatics:		-707.1108989715576 kJ / mol
+
```

Comment:
```suggestion

(Note that all of these outputs will have a `None` for their nonbonded energy, since the underlying engines have separated this out into `vdW` and `Electrostatics` components already)

```